odbc/src/api/functions.rs

```diff
@@ -1922,6 +1922,17 @@ macro_rules! sql_get_info_helper {
                         string_length_ptr,
                     )
                 }
+                InfoType::SQL_OJ_CAPABILITIES => {
+                    const OJ_CAPABILITIES: u32 = SQL_OJ_LEFT
+                      | SQL_OJ_NOT_ORDERED
+                      | SQL_OJ_INNER
+                      | SQL_OJ_ALL_COMPARISON_OPS;
+                    i16_len::set_output_fixed_data(
+                        &OJ_CAPABILITIES,
+                        info_value_ptr,
+                        string_length_ptr,
+                    )
+                }
                 InfoType::SQL_CATALOG_NAME_SEPARATOR => {
                     // The name separator used by MongoSQL is '.'.
                     i16_len::set_output_wstring_as_bytes(
```

Comment:
These are new functions that we either incorrectly reported that we did not support, or that we added support to mongosql during the DQ epic